src/main/java/seedu/address/model/patient/Event.java

```diff
@@ -12,9 +12,11 @@
  * Represents Events for a Patient
  */
 public class Event implements Comparable<Event> {
-    public static final String MESSAGE_CONSTRAINTS =
-            "Dates should be in the format: DD-MM-YYYY, HH:mm - HH:mm, OR if there is no time period,"
-            + "in the format: DD-MM-YYYY";
+    public static final String NAME_MESSAGE_CONSTAINTS = "The Name of the Event should be alphanumerical and non-empty";
+    public static final String DATETIME_MESSAGE_CONSTRAINTS = "Dates should be in the format: DD-MM-YYYY, "
+            + "HH:mm - HH:mm, where the End Time is after the Start Time, \n"
+            + "OR if there is no time period, in the format: DD-MM-YYYY";
+    public static final String NAME_PATTERN = "^.*[^a-zA-Z0-9 ].*$";
```

Comment:
`[^a-zA-Z0-9]` : is the not operator inside this set intended?

Reply:
oops, nice catch!